tests/page-objects/forms/generic-form.wdio.page.js

```diff
@@ -8,7 +11,31 @@ const nextPage = async (numberOfPages = 1) => {
   }
 };
 
+const formByHref = (href) => {
+  const css = `.action-container .general-actions .dropup.open .dropdown-menu li a[href="#/reports/add/${href}"]`;
+  return $(css);
+};
+
+const selectForm = async (formId, nonAdminUser = false) => {
```

Comment:
We already have an `openForm` method exposed in reports.wdio.page which we could use instead and not have duplicated logic.